medusa/indexers/tvmaze/tvmaze_api.py

```diff
@@ -371,7 +371,7 @@ def _parse_actors(self, tvmaze_id):
         for order, cur_actor in enumerate(actors.people):
             save_actor = Actor()
             save_actor['id'] = cur_actor.id
-            save_actor['image'] = cur_actor.image.get('original')
+            save_actor['image'] = cur_actor.image.get('original') if cur_actor.image else ''
```

Comment:
Also if you wanted to make sure save_actor['image'] was an empty string when non-existent and not `None`, you can do the following:  `cur_actor.image.get('original', '')`  or `cur_actor.image.get('original') or ''`.   Lastly you can also chain conditions such as `cur_actor.image.get('medium') or cur_actor.image.get('original') or ''`